### ISSUE TITLE
GFM Tables crash the pipeline

### ISSUE THREAD

**Issue by koraa** [adobe/helix-pipeline#158]
##  Tables

Github Flavourd Markdown Tables crash the markdown pipeline

```
# Foo
##  Tables

Github Flavourd Markdown Tables crash the markdown pipeline

```
# Foo

Hello World.

| foo | bar |
| --- | --- |
| baz | bim |
```

Hello World.

| foo | bar |
| --- | --- |
| baz | bim |
```

**Comment by koraa** [comment]
Possibly related: #156

**Comment by koraa** [comment]
Fixed with 4a4546fc0b5f0f3b365892bd30b57253236f3ef3
